js/currencycom.js

```diff
@@ -295,9 +303,9 @@ module.exports = class currencycom extends Exchange {
                     limitPriceMax = maxPrice;
                 }
             }
-            let precisionAmount = 1 / Math.pow (1, this.safeInteger (market, 'baseAssetPrecision'));
+            let precisionAmount = Precise.stringDiv ('1', this.parsePrecision (this.safeString (market, 'baseAssetPrecision')));
             let limitAmount = {
-                'min': Math.pow (10, -precisionAmount),
+                'min': this.parsePrecision ('-' + precisionAmount),
```

Comment:
I think we don't need to prepend the negative sign here, cause it will be negated in parsePrecision.